menus/atom-dart.cson

```diff
@@ -36,6 +36,8 @@
     {label: 'Format', command: 'dartlang:dart-format' }
     {label: 'Organize Directives', command: 'dartlang:organize-directives' }
     {label: 'Rename…', command: 'dartlang:refactor-rename' }
+    # TODO(danrubel) should this be placed in a Refactor submenu ?
```

Comment:
+1, or break the menu items up into those that modify code, and ones that are more information. I'll open an issue to track this.